package.json

```diff
@@ -182,7 +183,8 @@
     "workbox-build": "^3.6.3"
   },
   "resolutions": {
-    "**/hoist-non-react-statics": "^3.2.1"
+    "**/hoist-non-react-statics": "^3.2.1",
+    "jss": "^10.0.0-alpha.3"
```

Comment:
@oliviertassinari What was the reason for this? This means we only test with an alpha version of jss not the latest stable.

Reply:
Without this resolution, yarn will install JSS v10 and v9 at the same time. It breaks at runtime.